src/main/java/com/roisoftstudio/godutch/login/paths/SignPath.java

```diff
@@ -37,38 +40,45 @@ public Response getProtected() {
 
 
     @PUT
-    @Produces(APPLICATION_JSON)
-    @Consumes(APPLICATION_JSON)
     @Path("/up")
     public Response signUp(@NotNull Credentials credentials) {
         checkNotNull(credentials.getEmail(), "email"); // try to make work @NotNull annotation on param
         checkNotNull(credentials.getPassword(), "password");
         try {
             signService.signUp(credentials.getEmail(), credentials.getPassword());
-            return Response.ok("Account created successfully.").status(CREATED).build();
-        } catch (SignServiceException e) {
+          /*
+            Now returns this {status: CREATED, message:"Account created..."}
+            But i think its better return only HTTP code if its OK
+            And if an error has occurred, return HTTP + message (with ResponseError)
+          */
+            return Response
+                    .ok(new ResponseSucceed("Account created successfully.", CREATED))
```

Comment:
The intention of this message is for the people who will use the API it self, I think as a user of the API i like to read easily what happened with a nice text.